<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateFolderWithKeys</class>
 <widget class="QDialog" name="CreateFolderWithKeys">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create Media Folder</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../assets/icons/logic_icon16x16.png</normaloff>../assets/icons/logic_icon16x16.png</iconset>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>281</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(193, 211, 254);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>361</width>
      <height>267</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="fileLocationLabel">
         <property name="text">
          <string>File Location</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLineEdit" name="FileLocationLineEdit">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(237, 242, 251);</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="FileBrowseButton">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(215, 227, 252);</string>
           </property>
           <property name="text">
            <string>Browse</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QCheckBox" name="imageTypeCheckBox">
         <property name="text">
          <string>Image</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="videoTypeCheckBox">
         <property name="text">
          <string>Video</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="soundTypeCheckBox">
         <property name="text">
          <string>Sound</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="keywordLabel">
         <property name="text">
          <string>Keyword</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="cumImagesLabel">
         <property name="text">
          <string>Number Of Images</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLineEdit" name="keywordLineEdit">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(237, 242, 251);</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="numImgLineEdit">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(237, 242, 251);</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="createMediaButton">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(215, 227, 252);</string>
       </property>
       <property name="text">
        <string>Create Media</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="mediaProgressBar">
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="creationAction">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>200</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>350</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>350</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
